blocitoff/spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
```

Comment:
@jboltz88 All the auto-generated 3 specs below can be safely removed as they're not testing anything specific to the app.